CHANGELOG.md

```diff
@@ -2,6 +2,34 @@
 
 All notable changes to this project will be documented in this file.
 
+## Unreleased - as of Sprint 17 end 2015-01-05
+
+### [Added](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+17+Ending+Jan+5%2C+2015%22+label%3Aenhancement)
+
+- Rest API: Added VM Management Actions
+- UI  
+  - Can now set the locale for both server and user
+  - Repository Editor using AngularJS
+- Dynamic radio button support in dialogs
+- IPv6 support for literals
```

Comment:
Only the underlying libraries for vmware and rhevm/ovirt have been completed to support IPv6 literals with the latter still requiring a new release from rest-client and a new ruby 2.0 release.  Openstack, Amazon, Microsoft, are all TODO.

Additionally, the UI/backend modeling has not yet changed to allow only one of: ipv6/ipv4/hostname

Not sure how we want to word this:
`Underlying ruby/library support to communicate with rhevm/ovirt using IPv6 literal addresses`